### ISSUE TITLE
Tooltips

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#4101]
**Is your feature request related to a problem? Please describe.**
VIVO should be aligned with AA level of WCAG regarding **tooltips**.
